src/sql/parts/modelComponents/formContainer.component.ts

```diff
@@ -113,6 +116,16 @@ export default class FormContainer extends ContainerBase<FormItemLayout> impleme
 		return this._alignContent;
 	}
 
+	private getFormWidth(item: FormItem): string {
+		let itemConfig = item.config;
+		return itemConfig && itemConfig.width ? +itemConfig.width + 'px' : '400px';
+	}
+
+	private getComponentWidth(item: FormItem): string {
+		let itemConfig = item.config;
+		return itemConfig ? +itemConfig.componentWidth + 'px' : '';
```

Comment:
what's the purpose of the `+` in from of `itemConfig`?

Reply:
converting number to string

---
In reply to: [186891249](https://github.com/Microsoft/sqlopsstudio/pull/1371#discussion_r186891249) [](ancestors = 186891249)

Reply:
shouldn't the `+ 'px'` force the string conversion?  or is there some case I'm missing?

Reply:
it does, I'll fix it

---
In reply to: [186892512](https://github.com/Microsoft/sqlopsstudio/pull/1371#discussion_r186892512) [](ancestors = 186892512)